intake_elasticsearch/elasticsearch_seq.py

```diff
@@ -85,12 +93,12 @@ def _get_schema(self, retry=2):
                            npartitions=1,
                            extra_metadata={})
 
-    def _get_partition(self, _):
+    def _get_partition(self, _, slice_id=None, slice_max=None):
```

Comment:
The convention for the API of this function is `_get_partition(part)`, so in this case, `part` would be None in the case of no partitioning or (slice_id, slice_max) for parallel.

Reply:
Ok I will pass it as a tuple of the two values.